src/browser-frontend/selectors/ui-pages-selectors.js

```diff
@@ -44,8 +44,16 @@ export function getPageTabState(state, pageId) {
   return getPageVisuals(state, pageId).get('tabState');
 }
 
+export function getOptionalClass(state, pageId) {
+  return getPageVisuals(state, pageId).get('optionalClass');
+}
+
 // UI page computed properties getters.
 
+export function getPageDisplayOwner(state, pageId) {
+  return !!PagesSelectors.getPageOwnerId(state, pageId);
```

Comment:
You wouldn't need to negate here if you avoid the `|| ''` as suggested above.

Nit: Why have two selectors, both a `getPageOwnerId`, and a `getPageDisplayOwner`? Wouldn't the former be sufficient? I'm not saying this is necessarily wrong, but have you thought about some future where a "display owner" wouldn't be the "tab owner"?